code/modules/projectiles/ammunition/ego_ammunition/aleph.dm

```diff
@@ -9,3 +9,8 @@
 		var/mob/living/carbon/human/H = user
 		var/damage_mod = max(0.8, (H.sanityhealth / 100)) // Maximum SP will double the damage
 		BB.damage *= damage_mod
+
+/obj/item/ammo_casing/caseless/ego_adoration
+	name = "adoration slug"
+	desc = "A adoration slug."
+	projectile_type = /obj/projectile/melting_blob
```

Comment:
Projectile not under ego_bullet type.

Reply:
I see that you use the same projectile as the abnormality does, but I'd suggest making a new type for it just for balancing if we need them to deal different damage.